packages/caliper-core/lib/worker/caliper-worker.js

```diff
@@ -84,55 +85,56 @@ class CaliperWorker {
      * @param {Object} rateController rate controller object
      * @async
      */
-    async runFixedNumber(workloadModule, number, rateController) {
+    async _runFixedNumber(workloadModule, number, rateController) {
         const stats = this.internalTxObserver.getCurrentStatistics();
-        let error = undefined;
-        while (stats.getTotalSubmittedTx() < number && !error) {
+        const errors = [];
```

Comment:
Sorry my mistake, it looks like the worker terminates it's run at the first error. Apologies I missed that. It means we don't need to make changes around how errors are handled

Reply:
"I believe this can be made configurable with a flag—perhaps called **"AbortEarly"**—so that it defaults to the previous behavior. I think this approach also has the benefit of providing an overview of all errors during the initial run. What do you think?"

Reply:
It's a possibility, but I wonder where the control for this should be here. Unfortunately there is no documented behaviour for what submitTransaction will do if it throws an error so we have to assume that the current behaviour should be that the workload module decides whether to "Abort Early" or not given that it can catch the errors and throw one if neccesary to abort. All the examples will just let errors from the connector bubble up as they are very simple examples so nothing demonstrated there either.  So technically the ability to "Abort Early" or not already exists and can be controlled in the workload module so I'm not sure we need a flag as well. Sorry I'm not the author of this code and it's not an area I am familiar with as it existed before I started working on Caliper, so I would think the best person to advise on this is Attila so I would say you should raise an issue on github with this suggestion and then let Attila and I know it exists so that we can all discuss this properly. Sorry, I don't think it should be part of this PR so this PR should not be changing the behaviour so we need to revert this change.

Reply:
True that sounds like a completely different scope